tests/Decred/TWAnySignerTests.cpp

```diff
@@ -103,7 +103,7 @@ TEST(TWAnySignerDecred, PlanAndSign) {
     ANY_SIGN(input, TWCoinTypeDecred);
 
     ASSERT_TRUE(output.error().empty());
-    ASSERT_EQ(hex(output.encoded()), "0100000001fdbfe9dd703f306794a467f175be5bd9748a7925033ea1cf9889d7cf4dd1155000000000000000000002809698000000000000001976a914989b1aecabf1c24e213cc0f2d8a22ffee25dd4e188ace23bc8010000000000001976a9142a194fc92e27fef9cc2b057bc9060c580cbb484888ac000000000000000001000000000000000000000000ffffffff6a47304402203e6ee9e16d6bc36bb4242f7a4cac333a1c2a150ea16143412b88b721f6ae16bf02201019affdf815a5c22e4b0fb7e4685c4707094922d6a41354f9055d3bb0f26e630121026cc34b92cefb3a4537b3edb0b6044c04af27c01583c577823ecc69a9a21119b6");
+    ASSERT_EQ(hex(output.encoded()), "0100000001fdbfe9dd703f306794a467f175be5bd9748a7925033ea1cf9889d7cf4dd1155000000000000000000002809698000000000000001976a914989b1aecabf1c24e213cc0f2d8a22ffee25dd4e188ace03cc8010000000000001976a9142a194fc92e27fef9cc2b057bc9060c580cbb484888ac000000000000000001000000000000000000000000ffffffff6b483045022100ddbbbee3251a40718d7f4f45a400c87531b59469e423b39c205550932d4ad0e6022067182f0b135e4ff939ec408d80970e2de65d72257c8d3ec1c4e9eab7b76e7b8d0121026cc34b92cefb3a4537b3edb0b6044c04af27c01583c577823ecc69a9a21119b6");
```

Comment:
I don't think Decred supports segwit, should not be changed

Reply:
Decred has been also reverted, no change in behaviour.